DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/CentreAdministratorsViewModel.cs

```diff
@@ -28,7 +28,7 @@ int page
             MatchingSearchResults = filteredItems.Count;
             SetTotalPages();
             var paginatedItems = GetItemsOnCurrentPage(filteredItems);
-            var returnPage = string.IsNullOrEmpty(searchString) ? 1 : page;
+            var returnPage = searchString == null || string.IsNullOrEmpty(searchString.Trim()) ? page : 1;

```

Comment:
Could just simplify this to:
`var returnPage = string.IsNullOrWhitespace(searchString) ? page : 1;`